artcommon/artcommonlib/konflux/konflux_db.py

```diff
@@ -224,7 +224,7 @@ async def get_latest_build(
         :param name: component name, e.g. 'ironic'
         :param group: e.g. 'openshift-4.18'
         :param outcome: 'success' | 'failure'
-        :param assembly: non-standard assembly name, defaults to 'stream' if omitted
+        :param assembly: assembly name, if omitted any assembly is matched
```

Comment:
Hmm this behavior might do unexpected things, wondering why change this?

Reply:
That is required to mimic the behavior of [metadata.get_latest_build](https://github.com/openshift-eng/art-tools/blob/main/doozer/doozerlib/metadata.py#L417). Always defaulting to `stream` would make this impossible. I've also checked that we always pass in the assembly param when using the konflux_db function directly, so I don't see any risk in here.